src/Simplex/Messaging/Agent/Store/SQLite.hs

```diff
@@ -2248,7 +2249,7 @@ createWithRandomId' gVar create = tryCreate 3
           | otherwise -> pure . Left . SEInternal $ bshow e
 
 randomId :: TVar ChaChaDRG -> Int -> IO ByteString
-randomId gVar n = atomically $ U.encode <$> C.randomBytes n gVar
+randomId gVar n = atomically $ extractBase64 . U.encodeBase64' <$> C.randomBytes n gVar
```

Comment:
url format allows being unpadded for any size, but it breaks entity-encoding expectations elsewhere.